packages/app-admin-ui/client/providers/Session.js

```diff
@@ -1,4 +1,4 @@
-import { withApollo } from 'react-apollo';
+import { withApollo } from '@apollo/react-hoc';
```

Comment:
this is the only place `withApollo` is being used inside admin-ui code. once session is converted into functional component we can get rid of this as well and use standard query and mutation from hooks.